File: Nodo.py
class Nodo:
    
    def __init__(self,nombre,cualidad):
        self.nombre = nombre
        self.cualidad = cualidad
        self.nodoIzquierdo = None
        self.nodoDerecho = None

    def getNombre(self):
        return self.nombre

    def getCualidad(self):
        return self.cualidad

    def setNodoIzquierdo(self,nodo):
        self.nodoIzquierdo = nodo

    def getNodoIzquierdo(self):
        return self.nodoIzquierdo

    def setNodoDerecho(self,nodo):
        self.nodoDerecho = nodo
    
    def getNodoDerecho(self):
        return self.nodoDerecho
    
    

File: src/akinator/Nodo.java
package akinator;

public class Nodo {

    private String nombre;
    private String cualidad;
    private Nodo nodoIzquierdo;
    private Nodo nodoDerecho;

    public Nodo(String nombre, String cualidad) {
        this.nombre = nombre;
        this.cualidad = cualidad;
    }

    public String getNombre() {
        return this.nombre;
    }

    public String getCualidad() {
        return this.cualidad;
    }

    public void setNodoIzquierdo(Nodo nodo) {
        this.nodoIzquierdo = nodo;
    }

    public void setNodoDerecho(Nodo nodo) {
        this.nodoDerecho = nodo;
    }

    public Nodo getNodoIzquierdo() {
        return this.nodoIzquierdo;
    }

    public Nodo getNodoDerecho() {
        return this.nodoDerecho;
    }
}
